ComfortZoneApp/app/src/main/java/com/example/comfortzone/utils/WeatherDbUtil.java

```diff
@@ -67,4 +68,9 @@ public static void saveCities(WeatherGroupData weathers) {
             db.weatherDao().updateWeatherData(weatherFromDb);
         }
     }
+
+    public static List<WeatherData> getAll() {
+        return db.weatherDao().getAll();
```

Comment:
hmm, could we instead create a new SQL that gets only x number of items? seems like getting all of them then getting sublist of it is wasteful, unless you are just going to reuse the list